### PR TITLE
Added example folder with example files

### PR DESCRIPTION
**Pull request by dpear**
Previous PR should have changed setup.py to include importing the example data. Previous PR should have `testdata` folder and `example` for the tutorial data. This PR only includes the `example` folder files that are referenced in the README.

### CODE REVIEW

**Comment by dpear** [comment]
The estimating-biomass action now requires a table, same as read-count-threshold, instead of a total read count column. Now, the total reads for each sample are calculated from the input table. The README has also been updated to include an example that can be followed from start to finish that uses the data from the FMP as an example.

@antgonza I have verified that I get the same results as Jake when running this version of katharoseq. As a last check, Jake will run the plugin on his data and then we will be ready to resubmit the paper once the results are verified.

File: README.md [hunk 1]
```diff
@@ -14,35 +14,61 @@ pip install -e .
...
+Computation assumes that the user has classified their 16S features against SILVA, and that the `FeatureTable[Frequency]` has been collapsed to the genus level. Please see the [`q2-feature-classifier`](https://docs.qiime2.org/2021.11/plugins/available/feature-classifier/classify-sklearn) for detail on how to perform taxonomy classification, and the [`q2-taxa`](https://docs.qiime2.org/2021.11/plugins/available/taxa/collapse) plugin for information on collapsing to a taxonomic level. If you need more information on how to process your data, please refer to one of the relevant tutorials that can be found [here](https://docs.qiime2.org/2022.2/tutorials/). For these examples, data from the Fish Microbiome Project (FMP): [Fish microbiomes 101: disentangling the rules governing marine fish mucosal microbiomes across 101 species](https://www.biorxiv.org/content/10.1101/2022.03.07.483203v1) paper will be used, and can be found in the `example` folder.
```
Comment (antgonza) [inline README.md]: Can you update the links to qiime2 from 2021.11 to 2022.2?

**Review by antgonza — COMMENTED** [review]
Extra from fixing the tests, one comment. Thank you @dpear !

File: README.md (hunk 1 above)
Comment (dpear) [reply README.md]: @antgonza Yes, thanks. I've just pushed a new version with the `q2-feature-classifier` and `q2-taxa` links updated!

**Comment by antgonza** [comment]
Thank you @dpear !

**Pull request merged**